src/components/Text.js

```diff
@@ -64,3 +85,25 @@ const q5 = { id: 4, question: 'Are you an introvert or extrovert? How do you thi
 const q6 = { id: 5, question: 'Write a custom message' };
 const questions = [q1, q2, q3, q4, q5, q6];
 export const SelfQuestionList = questions;
+
+export const AboutText = () => {
+  return (
+    <font color="#FFFEE9">
+    <div>
+      <font size="7">
+        <p>Why was For Future Reference <i>created?</i>
+        </p>
+      </font>
+      <font size="4">
+        <p>For Future Reference started off as an idea between 4 friends who wanted to find a way to preserve our self reflections from the year 2020.
+        </p>
+        <p>2020 is unique because the entire world is discovering that some things are truly beyond our control. The more free time we have, the more
+        time we have to think and reflect on our lives and where we’re headed towards.
+        </p>
+        <p>For Future Reference is a tool that helps people to reflect and treasure relationships with one another by sending out emails to oneself / loved ones into the future.
+        </p>
+      </font>
+    </div>
+    </font>
+  );
+};
```

Comment:
This file should just be for text constants (shouldn't include any styling like color or font size, etc). So just like large sections of text so it doesn't clutter the rest of the code. My bad tbh for setting a bad/confusing example with the welcome text that I forgot I left in there. The best way to do this is probably to move the "Why was for Future Reference created" part into About.js and then just have a div that contains the 3 paragraphs that you export as `const AboutText` 
All the styling can go in `AboutStyles.js` (not sure if that files exists, but you can create it if not)

Reply:
if the styling is super minimal you can also do it inline or with a js object like this:
https://www.w3schools.com/react/react_css.asp